resources/README.md

```diff
@@ -0,0 +1,76 @@
+# Eventing-kafka Helm Chart
+
+This helm chart installs the eventing-kafka knative eventing implementation.  When installing make sure to provide the appropriate values file for the
+cluster you are deploying to.
+
+1. Remove the existing installation `helm delete --purge --tls eventing-kafka`
+1. Install the helm chart `helm install --tls -n eventing-kafka ./eventing-kafka -f <your value overrides>.yaml`
+
+## Kafka Providers
+
+Knative Kafka supports a number of kafka providers to configure a particular provider to be used, set the following
+in the values file:
+
+`environment.kafkaProvider: <CHOSEN PROVIDER>`
+
+The current allowed values are:
+
+* `local`: Standard Kafka installation with no special authorization required
+* `confluent`: Confluent Cloud
+* `azure`: Azure Event Hubs
+
+The provider chosen effects how authentication as well as admin calls (topic creation, deletion etc) work.
+
+## Credentials
+
+### Install & Label Kafka Credentials In Knative-Eventing Namespace
+
+Eventing-kafka depends on secrets labeled with `knativekafka.kyma-project.io/kafka-secret="true"`, multiple
```

Comment:
change to knative.dev?

Reply:
done, I missed one search term in my original commits :) all are changed now